src/pages/home/sidebar/AllSettingsScreen.tsx

```diff
@@ -65,7 +65,7 @@ function AllSettingsScreen({policies}: AllSettingsScreenProps) {
                   ]
                 : []),
             {
-                translationKey: 'allSettingsScreen.cardsAndDomains',
+                translationKey: 'allSettingsScreen.domains',
                 icon: Expensicons.CardsAndDomains,
```

Comment:
Should we also rename the icon? Or does it still have "cards" in it? 😅

Reply:
Haha yeah it has a card in it
<img width="40" alt="image" src="https://github.com/user-attachments/assets/e7e84d1e-90c3-4bc0-a81a-8a672c5f9e81">

Reply:
We should probably change the icon later?

Reply:
aah minteresting, yeah i would assume we'd want a different icon IFF we don't want to associate this item with cards anymore :D

Reply:
Yeahh, up to you Vit, feel free to self-merge if you wanna address the icon later

Reply:
Yep, I have asked about it here https://github.com/Expensify/App/issues/45021#issuecomment-2276847098 before, but opened this up for a review earlier as I considered it NAB

Reply:
Pushed using the globe icon 
<img width="383" alt="image" src="https://github.com/user-attachments/assets/8528a87d-dbe0-43e2-80f2-17edba3eb874">